src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/Generated/Recipe.cs

```diff
@@ -214,6 +214,27 @@ private void ConfigureBeanstalkEnvironment(Configuration settings)
                 });
             }
 
+            if (!string.IsNullOrEmpty(settings.HealthCheckURL))
```

Comment:
We should be extra defensive and only set health check url if doing a load balanced deployment. Don't just rely on the recipe `depends` criteria.

Reply:
Done